### PR TITLE
chore: rename prisma-fmt-wasm to prisma-schema-wasm

### PR DESCRIPTION
**Pull request by Jolg42**
Part of https://github.com/prisma/prisma-engines/pull/4033

### CODE REVIEW

**Comment by Jolg42** [comment]
Note: ignore the "The system cannot find the path specified." for Windows run, it looks like it started with https://github.com/prisma/prisma/pull/19427 and I'm checking what is the problem there.

File: packages/internals/src/wasm.ts [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
+import prismaSchemaWasm from '@prisma/prisma-schema-wasm'
 
 import { WasmPanicRegistry } from './WasmPanicRegistry'
 
 // Note: using `import { dependencies } from '../package.json'` here would break esbuild with seemingly unrelated errors.
 // eslint-disable-next-line @typescript-eslint/no-var-requires
 const { dependencies } = require('../package.json')
 
-export { prismaFmt }
+export { prismaSchemaWasm }
```
Comment (jkomyno) [inline packages/internals/src/wasm.ts:9]: Nit: do we really need a named export in a `wasm.ts` file to have a `-Wasm` postfix?

**Review by jkomyno — APPROVED** [review]
It seems that both the CI and the codebase are handled! Nice, approving with a small nit that doesn't block merges.

File: packages/internals/src/wasm.ts (hunk 1 above)
Comment (Jolg42) [reply packages/internals/src/wasm.ts:9]: `prismaSchema` is usually used in the codebase to refer to the Prisma schema file, so I think we can be very explicit here

File: packages/internals/src/__tests__/engine-commands/formatSchema.test.ts [hunk 2]
```diff
@@ -12,7 +12,7 @@ if (process.env.CI) {
 
 const ctx = jestContext.new().add(jestConsoleContext()).assemble()
 
-describe('format wasm', () => {
+describe('schema wasm', () => {
```
Comment (janpio) [inline packages/internals/src/__tests__/engine-commands/formatSchema.test.ts:15]: doesn't this refer to the action of formatting?

File: packages/internals/src/panic.ts [hunk 3]
```diff
@@ -36,6 +36,7 @@ export enum ErrorArea {
   PHOTON_STUDIO = 'PHOTON_STUDIO',
   // Unused since 4.9.0 and now using `LIFT_CLI`
   INTROSPECTION_CLI = 'INTROSPECTION_CLI',
+  // Note: prisma-fmt-wasm was renamed to prisma-schema-wasm in 5.0.0
```
Comment (janpio) [inline packages/internals/src/panic.ts:39]: What does this refer to here?

File: packages/internals/src/wasm.ts (hunk 1 above)
Comment (janpio) [reply packages/internals/src/wasm.ts:9]: I agree, just `prismaSchema` would be dangerous.

File: packages/internals/src/wasm.ts [hunk 4]
```diff
@@ -1,14 +1,14 @@
...
 // e.g. 4.3.0-18.a39215673171b87177b86233206a5d65f2558857
-export const prismaFmtVersion: string = dependencies['@prisma/prisma-fmt-wasm']
+export const prismaSchemaVersion: string = dependencies['@prisma/prisma-schema-wasm']
```
Comment (janpio) [inline packages/internals/src/wasm.ts:11]: Shouldn't this be `prismaSchemaWasmVersion` then?

Comment (jkomyno) [reply packages/internals/src/wasm.ts:11]: Right, https://github.com/prisma/prisma/pull/20106

File: packages/internals/src/panic.ts (hunk 3 above)
Comment (jkomyno) [reply packages/internals/src/panic.ts:39]: The error area is something used in our internal error reporting dashboard to categorise Rust panics' "topic areas". It mirrors this `ErrorArea` enum [here](https://github.com/prisma/error-handling-backend/blob/482bf538ec055f4ef41926736d72c5f08e3b9a07/src/types/enums/index.ts#L11-L27).

File: packages/internals/src/__tests__/engine-commands/formatSchema.test.ts (hunk 2 above)
Comment (Jolg42) [reply packages/internals/src/__tests__/engine-commands/formatSchema.test.ts:15]: Yes, here actually a better name would be "format" but actually these tests are now duplicates of the ones below, and we could clean up there, though not urgent at all.

**Pull request merged**